pulpcore/tests/unit/test_settings.py

```diff
@@ -1,62 +1,64 @@
-from unittest import TestCase, mock
+# TODO: Implement a dynaconf test case here
```

Comment:
We've planned to add tests later in many areas so I'm ok to remove this with that same thinking. Otherwise someone will need to add them before we can merge. @rochacbruno what should we do?